adapters/mqtt-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2023 Contributors to the Eclipse Foundation
```

Comment:
The Eclipse Foundation does not require us to update this anymore. So no need to change the comment ...

Reply:
I reverted similar change in all files I modify now.